src/proto/mgmt/pool.proto

```diff
@@ -87,3 +87,44 @@ message ListContResp {
 	int32 status = 1; // DAOS error code
 	repeated Cont containers = 2; // containers
 }
+
+// PoolQueryReq represents a pool query request.
+message PoolQueryReq {
+	string uuid = 1;
+}
+
+// StorageUsageStats represents usage statistics for a storage subsystem.
+message StorageUsageStats {
+	uint64 total = 1;
+	uint64 free = 2;
+	uint64 min = 3;
+	uint64 max = 4;
+	uint64 mean = 5;
+}
+
+// PoolRebuildState indicates the pool's rebuild state.
+enum PoolRebuildState {
```

Comment:
could this be embedded inside PoolRebuildStatus, might be neater given it's only used there

Reply:
Didn't occur to me that I could do that, but I see some precedent in the JoinResp message. I'll fix this in a follow-up PR.